WORKSPACE

```diff
@@ -253,6 +253,10 @@ maven_install(
             group = "com.google.apis",
             version = "v1-rev235-1.25.0",
         ),
+        # https://github.com/bazelbuild/rules_jvm_external/issues/907
+        # Any two platforms to ensure that it doesn't work _only_ under the host operating system
```

Comment:
Excellent. Thank you

Reply:
I do thank _you_, @shs96c, knowing how time-consuming code reviews can be...